src/pages/index.tsx

```diff
@@ -1,5 +1,10 @@
+import ProductList from '@/components/ProductList';
 const Home = () => {
-  return <div></div>;
+  return (
+    <>
+      <ProductList />
+    </>
```

Comment:
```suggestion
      <ProductList />
```
Request: 불필요한 Fragment는 삭제해주세요.

Reply:
삭제했습니다!